target/README.md

```diff
@@ -13,85 +13,113 @@
+| UserName | The UserName to connect to the API | Yes       |
+| Password | The Password to connect to the API | Yes       |
+| BaseUrl  | The URL to the API                 | Yes       |
+
+### Correlation configuration
 
-2. Specify the following configuration:
+The correlation configuration is used to specify which properties will be used to match an existing account within _{connectorName}_ to a person in _HelloID_.
 
-    | Setting                   | Value                             |
-    | ------------------------- | --------------------------------- |
-    | Enable correlation        | `True`                            |
-    | Person correlation field  | `PersonContext.Person.ExternalId` |
-    | Account correlation field | `EmployeeNumber`                  |
+| Setting                   | Value                             |
+| ------------------------- | --------------------------------- |
+| Enable correlation        | `True`                            |
+| Person correlation field  | `PersonContext.Person.ExternalId` |
+| Account correlation field | `EmployeeNumber`                  |
 
 > [!TIP]
 > _For more information on correlation, please refer to our correlation [documentation](https://docs.helloid.com/en/provisioning/target-systems/powershell-v2-target-systems/correlation.html) pages_.
 
-#### Field mapping
+### Available lifecycle actions
+
+The following lifecycle actions are available:
+
+| Action                                  | Description                                                                                 |
+| --------------------------------------- | ------------------------------------------------------------------------------------------- |
+| create.ps1                              | Creates a new account.                                                                      |
+| delete.ps1                              | Removes an existing account or entity.                                                     |
+| disable.ps1                             | Disables an account, preventing access without permanent removal.                           |
+| enable.ps1                              | Enables an account, granting access.                                                       |
+| update.ps1                              | Updates the attributes of an account.                                                      |
+| permissions/groups/grantPermission.ps1  | Grants specific permissions to a user or group.                                            |
+| permissions/groups/revokePermission.ps1 | Revokes specific permissions from a user or group.                                         |
+| permissions/groups/permissions.ps1      | Retrieves all available permissions.                                                       |
+| resources/groups/resources.ps1          | Manages resources, such as creating groups.                                                |
```

Comment:
Dit is nu meer een voorbeeld. Waar we bij de Create.ps1 precies aangeven wat er gebeurd. Nu moet je deze regel altijd nog aanpassen naar de gewenst situatie.
Ik zou het gelijk houden aan de schrijfwijze bij de create.ps1, dus zoiets van maken:
`Manages resources. Creates groups.`

Reply:
Ik laat 'm even zoals het nu is. Deze tekst moet toch altijd worden aangepast omdat niet altijd groepen maakt maar bv ook ou's (AD) of wat dan ook.